Woof/Woof/Model/Sitter/Sitter+DummyData.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+extension Sitter {
+    static let dummySitter = Sitter(
```

Comment:
nit:

```suggestion
    static let dummyEmilyDoe = Sitter(
```
?


I'd prefer to use it as:
```swift
let sitter = Sitter.dummyEmilyDoe
```

or

```swift
let sitter = Sitter.Dummy.EmilyDoe
```

Reply:
Fixed with https://github.com/ios-course/ironfoudation-team-project/pull/38/commits/e3cecfe033d1f4b3263e510ebfde6c655f755d29